### PR TITLE
jenkins settings were changed

### PR DESCRIPTION
**Pull request by Andreichenko**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -8,6 +8,9 @@ timestamps {
             checkout scm
         }
         stage('Build'){
+            steps {
+               sh 'mvn -B -DskipTests clean package'
+            }
```
Comment (freideveloper) [inline Jenkinsfile]: NO

Comment (Andreichenko) [reply Jenkinsfile]: I try to clarify this issue

File: Jenkinsfile [hunk 2]
```diff
@@ -26,6 +25,12 @@ timestamps {
             '''
 
         }
+        stage('Build'){
+
+         sh 'mvn -B -DskipTests clean package'
+
+        }
+
```
Comment (freideveloper) [inline Jenkinsfile:33]: OK, true

Comment (Andreichenko) [reply Jenkinsfile:33]: Thanks

**Pull request merged**